docs/en/faq.md

```diff
@@ -1,332 +1,169 @@
-# FAQs
+# Frequently Asked Questions
 
-## Why should I install a computing engine like Spark or Flink?
+## Do I need to install engines like Spark or Flink to use SeaTunnel?
+No, SeaTunnel supports Zeta, Spark, and Flink as options for the integration engine. You can choose one of them. The community especially recommends using Zeta, a new-generation high-performance engine specifically built for integration scenarios.
+The community provides the most support for Zeta, which also has richer features.
 
-SeaTunnel now uses computing engines such as Spark and Flink to complete resource scheduling and node communication, so we can focus on the ease of use of data synchronization and the development of high-performance components. But this is only temporary.
+## What data sources and destinations does SeaTunnel support?
+SeaTunnel supports a variety of data sources and destinations. You can find the detailed list on the official website:
+- Supported data sources (Source): https://seatunnel.apache.org/docs/connector-v2/source
+- Supported data destinations (Sink): https://seatunnel.apache.org/docs/connector-v2/sink
 
-## I have a question, and I cannot solve it by myself
+## Which data sources currently support CDC (Change Data Capture)?
+Currently, CDC is supported for MongoDB CDC, MySQL CDC, OpenGauss CDC, Oracle CDC, PostgreSQL CDC, SQL Server CDC, TiDB CDC, etc. For more details, refer to the [Source](https://seatunnel.apache.org/docs/connector-v2/source) documentation.
 
-I have encountered a problem when using SeaTunnel and I cannot solve it by myself. What should I do? First, search in [Issue List](https://github.com/apache/seatunnel/issues) or [Mailing List](https://lists.apache.org/list.html?dev@seatunnel.apache.org) to see if someone has already asked the same question and got an answer. If you cannot find an answer to your question, you can contact community members for help in [These Ways](https://github.com/apache/seatunnel#contact-us).
-
-## How do I declare a variable?
-
-Do you want to know how to declare a variable in SeaTunnel's configuration, and then dynamically replace the value of the variable at runtime?
-
-Since `v1.2.4`, SeaTunnel supports variable substitution in the configuration. This feature is often used for timing or non-timing offline processing to replace variables such as time and date. The usage is as follows:
-
-Configure the variable name in the configuration. Here is an example of sql transform (actually, anywhere in the configuration file the value in `'key = value'` can use the variable substitution):
+## Does it support CDC from MySQL replica? How is the log fetched?
+Yes, it is supported by subscribing to the MySQL binlog and parsing the binlog on the synchronization server.
```

Comment:
This part already had in https://seatunnel.apache.org/docs/2.3.8/connector-v2/source/MySQL-CDC#enabling-the-mysql-binlog